### ISSUE TITLE
Two filters block the Adult Swim app from starting on Roku

### ISSUE THREAD

**Issue by BlohoJo** [StevenBlack/hosts#2363]
This is for Roku, I don't know about other platforms.

Today the Adult Swim app/channel received an update on Roku. It will now no longer start up unless the follow two filters are unblocked:
```
medium.ngtv.io
turnip.cdn.turner.com
```

Currently working around this by using the following custom filtering rules in AdGuard:
```
@@||medium.ngtv.io^$important
@@||turnip.cdn.turner.com^$important
```

**Comment by hagezi** [comment]
These must also be unblocked for CNN videos: https://github.com/hagezi/dns-blocklists/issues/899

**Comment by StevenBlack** [comment]
Thank you @BlohoJo and thank you for the assist Gerd @hagezi.

This is fixed in commit [bbe11d6](https://github.com/StevenBlack/hosts/commit/bbe11d622e3317ff6b43927cbc211486c785e94b) and will be live in the next release.
